guides.md

```diff
@@ -501,20 +501,30 @@ members. Skip the steps below at your discretion for low-stakes topics._
   - as needed, send reminders of action items
   - migrate action items into task tracker
   
-### Joining a Jitsi Call
+### Joining a BigBlueButton Call
 
-💻 **Desktop.** Follow [`link.hypha.coop/calls`](https://link.hypha.coop/calls) to our default conference platform.
-
-📱 **Mobile**
-  1. Install the app. [Android][jitsi-android] | [iOS][jitsi-ios]
-  2. 🔗 **Visit** call URL to open app, or 📋 **copy-paste** redirected call URL (e.g. `https://jitsi.hypha.coop/hyphacoop`) into app home screen.
-  
-   [jitsi-android]: http://jitsi.org/android
-   [jitsi-ios]: http://jitsi.org/ios
+💻 **Desktop and 📱Mobile.** Follow [`link.hypha.coop/calls`](https://link.hypha.coop/calls) to our default conference platform.
 
```

Comment:
```suggestion

*Mic Selection* - With some browsers (such as FireFox) you will ask you for the mic selection when the browser asks you for permission to use the microphone. Other browsers (such as Chrome) will not.  In those cases you will have to proceed to the `Echo Test` and answer `No` to `Do you hear audio?`. This will bring up a dialog to change your mic input.
```